python3/packages/observer.py

```diff
@@ -371,6 +371,11 @@ def _patch_module(module_name):
     # If there are no configs, or an exception is raised, span and patch_module
     # are not overridden and will be the defined no-op functions.
     span, patch_module = _init_tracing(observer_configs, observer_config_dir)
+
+    # If tracing is now operational, explicity set "OTEL_SDK_DISABLED" to "false".
```

Comment:
"now" or "not"?

Reply:
The double negation is a bit annoying, but I wanted to follow the opentelemetry standard...